reactor-core/src/main/java/reactor/core/publisher/FluxMergeOrdered.java

```diff
@@ -76,23 +81,29 @@
 	 * This operation doesn't change the current {@link FluxMergeOrdered} instance.
 	 *
 	 * @param source the new source to merge with the others
+	 * @param delayError delaying errors until all data is consumed
 	 * @return the new {@link FluxMergeOrdered} instance
 	 */
 	FluxMergeOrdered<T> mergeAdditionalSource(Publisher<? extends T> source,
-			Comparator<? super T> otherComparator) {
+			Comparator<? super T> otherComparator,
+			boolean delayError) {
 		int n = sources.length;
 		@SuppressWarnings("unchecked")
 		Publisher<? extends T>[] newArray = new Publisher[n + 1];
 		System.arraycopy(sources, 0, newArray, 0, n);
 		newArray[n] = source;
 
+		if (delayError != this.delayError) {
```

Comment:
this shouldn't be a parameter / throwing, as discussed above. instead, use `this.delayError` for the new instance.